pos_default_quantity/views/product_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
name of the field should not be uom_category_view.xml instead ?

Reply:
it is `id="product_uom_categ_form_view"` in [12.0](https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/views/product_view.xml#L87)